modules/View.js

```diff
@@ -459,159 +545,101 @@
-     * @method detachSharedViews
+     *   @param [options.shared=false] {Boolean} If true, detach only the shared views. These are views not owned by this parent. As compared to a child view
+     *                                           which are disposed when the parent is disposed.
+     *   @param [options.child=false] {Boolean} If true, detach only child views. These are views that are owned by the parent and dispose of them if the parent is disposed.
+     * @method detachTrackedViews
      */
     detachTrackedViews: function(options) {
-      var trackedViewsHash = this.__getTrackedViewsHash(options);
+      var trackedViewsHash = this.getTrackedViews(options);
       _.each(trackedViewsHash, function(view) {
         view.detach();
       });
     },
 
     /**
-     * Detach all child views. NOTE: this is not recursive - it will not separate the entire view tree.
-     * @method detachChildViews
-     * @deprecated 0.3.x - use this.detachTrackedViews({ shared: false }); instead
-     */
-    detachChildViews: function() {
-      this.detachTrackedViews({ shared: false });
-    },
-
-    /**
-     * Activates all tracked views (either all shared or all child views based on options.shared).
-     *
+     * Activates all tracked views or a subset of them based on the options parameter.
      * @param [options={}] {Object}  Optional options.
-     *   @param [options.shared=false] {Boolean} The view is a shared view instead of a child view
-     *                                           (shared views are not disposed when the parent is disposed)
+     *   @param [options.shared=false] {Boolean} If true, activate only the shared views. These are views not owned by this parent. As compared to a child view
+     *                                           which are disposed when the parent is disposed.
+     *   @param [options.child=false] {Boolean} If true, activate only child views. These are views that are owned by the parent and dispose of them if the parent is disposed.
      * @method activateTrackedViews
      */
     activateTrackedViews: function(options) {
-      var trackedViewsHash = this.__getTrackedViewsHash(options);
+      var trackedViewsHash = this.getTrackedViews(options);
       _.each(trackedViewsHash, function(view) {
         view.activate();
       });
     },
 
     /**
-     * Activates all child views
-     * @method activateChildViews
-     * @deprecated 0.3.x - use this.activateTrackedViews({ shared: false }); instead
-     */
-    activateChildViews: function() {
-      this.activateTrackedViews({ shared: false });
-    },
-
-    /**
      * Binds the view as a tracked view - any recursive calls like activate, deactivate, or dispose will
-     * be done to the tracked view as well.  Except dispose for shared views.
-     *
+     * be done to the tracked view as well.  Except dispose for shared views. This method defaults to register the
```

Comment:
hard to know what the right order is. There are a lot of tracked view get/has/change methods. They are mostly bundled together already but again there are many of them. Also there's the organization of flow of rendering, or how common the method is.